llama_index/llms/base.py

```diff
@@ -76,17 +76,39 @@ def __str__(self) -> str:
 
 
 class LLMMetadata(BaseModel):
-    """LLM metadata."""
-
-    context_window: int = DEFAULT_CONTEXT_WINDOW
-    num_output: int = DEFAULT_NUM_OUTPUTS
-    is_chat_model: bool = False
-    is_function_calling_model: bool = False
-    # By default we don't know the model name. We can set it automatically for
-    # some types, but custom predictors (like locally loaded models) we won't
-    # know.
-    # Used for tests, logging, and sanity checks
-    model_name: str = "unknown"
+    context_window: int = Field(
+        default=DEFAULT_CONTEXT_WINDOW,
+        description="Number of tokens the model can intake when generating a response.",
+    )
+    num_output: int = Field(
+        default=DEFAULT_NUM_OUTPUTS,
+        description="Number of tokens the model can output when generating a response.",
+    )
+    is_chat_model: bool = Field(
+        default=False,
+        description=(
+            "Set True if the model supports chatting, like OpenAI's"
```

Comment:
nit: to be more precise, rather than "supports chatting", more about whether if the API exposes a chat interface (i.e. directly takes in a list of messages, rather than a raw text input)

Reply:
Appreciated the clarification, implemented!